Update receipt.query

timgdavies opened the pull request:
Update to add country name in brackets to Paid To listings when an entity is present.

timgdavies committed:
Update receipt.query

Update to add country name in brackets to Paid To listings when an entity is present.

caprenter commented:
For reference the problem to be addressed is here:
http://alpha.resourceprojects.org/project/PH/apma-l9nepr.html

Commit:
Alters the tests so that this change passes

The string (Philippines) should now appear in the payments table
on certain pages.

caprenter commented:
As long as this passes the tests, this is good to go.
